### PR TITLE
Fix error when adding a new named property in substation creation dialog

### PR DESCRIPTION
**Pull request by niconoir**

### CODE REVIEW

File: src/components/dialogs/substation-creation-dialog.js [hunk 1]
```diff
@@ -58,13 +60,18 @@ const NonNullStringPair = ({
...
+    };
     const [name, nameField] = useAutocompleteField({
         id: 'pairKey' + index,
         label: 'PropertyName',
         formProps: filledTextField,
         validation: { isFieldRequired: true },
         values: predefinedNames,
         allowNewValue: true,
+        getLabel: getIdOrSelf,
+        newEntryToValue: entryToValue,
```
Comment (laurgarn) [inline src/components/dialogs/substation-creation-dialog.js:74]: Just give func_identity for newEntryToValue

File: src/components/dialogs/substation-creation-dialog.js [hunk 2]
```diff
@@ -81,6 +88,8 @@ const NonNullStringPair = ({
         validation: { isFieldRequired: true },
         values: predefinedValues,
         allowNewValue: true,
+        getLabel: getIdOrSelf,
+        newEntryToValue: entryToValue,
```
Comment (laurgarn) [inline src/components/dialogs/substation-creation-dialog.js:92]: Same as above

File: src/components/dialogs/substation-creation-dialog.js (hunk 1 above)
Comment (niconoir) [reply src/components/dialogs/substation-creation-dialog.js]: Fixed

File: src/components/dialogs/substation-creation-dialog.js (hunk 2 above)
Comment (niconoir) [reply src/components/dialogs/substation-creation-dialog.js]: Fixed

**Review by laurgarn — APPROVED** [review]

**Review by etiennehomer — APPROVED** [review]

**Pull request merged**